### PR TITLE
[MRG] Item multi inheritance fix

### PR DESCRIPTION
**Pull request by nramirezuy**

### CODE REVIEW

**Comment by chekunkov** [comment]
What is missing in this PR? Can it be merged?

**Comment by dangra** [comment]
@chekunkov anything to comment on its implementation? 

From my past comments I still don't get a test case for defining and/or overriding fields of its parents.

**Comment by nramirezuy** [comment]
@dangra https://github.com/scrapy/scrapy/pull/353/files#diff-b610b6c1bd7114fdb73120bf028608caR169 ?

File: scrapy/item.py [hunk 1]
```diff
@@ -24,18 +24,20 @@ class Field(dict):
 class ItemMeta(ABCMeta):
 
     def __new__(mcs, class_name, bases, attrs):
+        _class = super(ItemMeta, mcs).__new__(mcs, 'x_' + class_name, tuple(base._class for base in bases if hasattr(base, '_class')), attrs)
```
Comment (kmike) [inline scrapy/item.py]: hey @nramirezuy - could you please split it into multiple lines? 
e.g. `tuple(base._class for base in bases if hasattr(base, '_class'))` is a good candidate for a local variable.

Comment (nramirezuy) [inline scrapy/item.py]: done :smile:

File: tests/test_item.py [hunk 2]
```diff
@@ -134,6 +134,69 @@ class TestItem(BaseItem):
...
+
+        class B(A): pass
+
+        class C(A):
+            save = Field(default='C')
+
+        class D(B, C): pass
+
+        self.assertEqual(D(save='X')['save'], 'X')
+        self.assertEqual(D.fields, {'save': {'default': 'C'}})
```
Comment (dangra) [inline tests/test_item.py]: I think it lacks another testcase where class D overrides a field defined in class A or B.

Comment (nramirezuy) [inline tests/test_item.py]: Done for D and E

**Comment by dangra** [comment]
You have my +1 to merge as long as tests passes in Travis-CI for python2 and python3

**Pull request merged**